evalml/automl/automl_search.py

```diff
@@ -505,6 +505,16 @@ def __init__(
             allowed_estimators = get_estimators(
                 self.problem_type, self.allowed_model_families
             )
+            if (
```

Comment:
I'm on the fence as to whether this should live here or in `get_estimators`. I'm ok keeping this here. Is this a bug that sktime can fix in the future or is it a limitation of the `MS` frequency?

Reply:
The only reason I chose to put it here instead of `get_estimators` was to avoid passing `problem_configuration` to `get_estimators` just for this check

Reply:
Yea let's not change the api of `get_estimators` yet! Ok, I think there will be other frequencies with that limitation so I will file an issue for following up about that. Might make sense to add yet another data check for this hehe